pkg/controller/container-runtime-config/container_runtime_config_controller_test.go

```diff
@@ -100,6 +103,15 @@ func newFixture(t *testing.T) *fixture {
 	return f
 }
 
+func newConfigMap(name string) *k8sv1.ConfigMap {
```

Comment:
hmm. I think this could be more descriptive and actually construct the config map we want to use.